vanta/table_vanta_monitor.go

```diff
@@ -0,0 +1,111 @@
+package vanta
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+	"github.com/turbot/steampipe-plugin-vanta/api"
+)
+
+//// TABLE DEFINITION
+
+func tableVantaMonitor(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "vanta_monitor",
+		Description: "Vanta Monitor",
+		List: &plugin.ListConfig{
+			Hydrate: listVantaMonitors,
+		},
+		Columns: []*plugin.Column{
+			{Name: "name", Type: proto.ColumnType_STRING, Description: "A human-readable name of the test."},
+			{Name: "id", Type: proto.ColumnType_STRING, Description: "A unique identifier of the test."},
+			{Name: "category", Type: proto.ColumnType_STRING, Description: "A high-level categorization of the test."},
+			{Name: "outcome", Type: proto.ColumnType_STRING, Description: "Outcome of the test run. Possible values are: 'PASS', 'DISABLED', 'FAIL', 'IN_PROGRESS', 'INVALID' and 'NA'."},
+			{Name: "test_id", Type: proto.ColumnType_STRING, Description: "A unique identifier for this test."},
```

Comment:
Is the description correct? This looks similar to the description of `id` column.